v3/plugins/eclipse/che-machine-exec-plugin/nightly/meta.yaml

```diff
@@ -27,3 +27,5 @@ spec:
      image: "quay.io/eclipse/che-machine-exec:nightly"
      ports:
        - exposedPort: 4444
+     command: ['/go/bin/che-machine-exec', '--static', '/cloud-shell', '--url', '127.0.0.1:4444']
```

Comment:
Do you plan to update cloud-shell as well where `quay.io/eclipse/che-machine-exec:nightly` is used
https://github.com/eclipse/che-plugin-registry/blob/master/v3/plugins/eclipse/cloud-shell/nightly/meta.yaml#L26 ?